README.md

```diff
@@ -81,6 +83,15 @@ Whereas for CLN nodes, the following information is required:
 }
 ```
 
+For Eclair, the following information is required:
+```
+{ 
+  "id": <node_id>,
+  "base_url": <scheme://ip:port or scheme://domain:port>,
```

Comment:
nit: remove `scheme://` here and `http:` below so that the readme has the simplest example possible

Reply:
Addressed: https://github.com/bitcoin-dev-project/sim-ln/pull/225/commits/be85f5b1fa6cacc54b1885a35d4445f963954323
I also did some light cleanup and moved list of implementations to be above the prerequisites (it's more important IMO). Let me know if any of these changes doesn't work for you.